gluoncv/utils/lr_scheduler.py

```diff
@@ -106,3 +111,109 @@ def update(self, i, epoch):
                     (1 + cos(pi * (T - self.warmup_N) / (self.N - self.warmup_N))) / 2
             else:
                 raise NotImplementedError
+
+class DistLRScheduler(lr_scheduler.LRScheduler):
```

Comment:
Is there a way to make this PR and https://github.com/dmlc/gluon-cv/pull/353 compatible?